config/locales/en.yml

```diff
@@ -974,6 +974,58 @@ en:
   community_id_is_precise: Community Taxon is precise
   community_moderation_settings: Community Moderation Settings
   community_supported_id: Community-supported Taxon?
+  compact_number_formatting:
+    x_thousand:
+      one: "%{count}K"
+      other: "%{count}K"
+    # Short notation representing a four-digit number, i.e. 1,000
+    x_ten_thousand:
+      one: "%{count}K"
+      other: "%{count}K"
+    # Short notation representing a five-digit number, i.e. 10,000
+    x_hundred_thousand:
+      one: "%{count}K"
+      other: "%{count}K"
+    # Short notation representing a six-digit number, i.e. 100,000
+    x_million:
+      one: "%{count}M"
+      other: "%{count}M"
+    # Short notation representing a seven-digit number, i.e. 1,000,000
+    x_ten_million:
+      one: "%{count}M"
+      other: "%{count}M"
+    # Short notation representing an eight-digit number, i.e. 10,000,000
+    x_hundred_million:
+      one: "%{count}M"
+      other: "%{count}M"
+    # Short notation representing a nine-digit number, i.e. 10,000,000
+    x_billion:
+      one: "%{count}B"
+      other: "%{count}B"
```

Comment:
I don't think I've ever seen a billion abbreviated as `B` in a chart. `G` for giga- seems much more common, as per https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes.